+sig/+node/Signal.m

```diff
@@ -308,31 +308,36 @@
     end
     
     function tr = applyTransferFun(varargin)
-      % New signal derived by applying a transfer function to input nodes
+      % New signal derived by applying a transfer function to input node(s)
+      % 
+      % This function creates a node from input values (which can be
+      % signals or non-signals), then creates a new signal containing the
+      % new node.
+      % 
+      % Inputs:
+      %   `varargin`: contains one (or more) input values/signals, `sigs`, 
+      %   used to create the output signal; a string, `funName`, of the
+      %   transfer function; an optional function handle, `funArg`, which
+      %   can be applied by the transfer function (e.g. the transfer
```

Comment:
Maybe remove the e.g. and instead make the two examples below concrete, like so:
```
rt = s1.applyTransfer(s2, 'mapn', '@plus', '%s.mapn(%s, %s)'
```

Reply:
Sure, good idea